twake/backend/node/src/services/channels/index.ts

```diff
@@ -6,7 +6,7 @@ import web from "./web/index";
 import { Channel } from "./entities";
 import { DatabaseServiceAPI } from "../../core/platform/services/database/api";
 
-@Prefix("/api/channels")
+@Prefix("/internal/services/channels/v1")
```

Comment:
I still do not understand why we need to change this. We can have this API at http://localhost:3000/api/channels. This is up to the reverse proxy to redirect http://twake:port/internal/services/channels/v1 to http://localhost:3000/api/channels